lib/ogr/geometry/surface_methods.rb

```diff
@@ -13,10 +15,17 @@ def area
 
       # Returns the units used by the associated OGR::SpatialReference.
       #
-      # @return [Hash]
+      # @return [{ unit_name: String, value: Float }, nil]
```

Comment:
Is this valid yard?

Reply:
```
ffi-gdal on  new-geom-types [$] via 💎 v3.0.1 took 2s
❯ yard
Files:         168
Modules:        77 (   69 undocumented)
Classes:       152 (  124 undocumented)
Constants:      86 (   71 undocumented)
Attributes:     16 (    0 undocumented)
Methods:       872 (   61 undocumented)
 72.98% documented
```
if it's not, `yard` doesn't complain about it.

Reply:
<img width="511" alt="image" src="https://user-images.githubusercontent.com/142010/128260306-89653db3-c6e1-4dc9-b5df-029807fb7600.png">
Looks alright...